cms/models.py

```diff
@@ -1142,6 +1145,10 @@ def save(self, clean=True, user=None, log_action=False, **kwargs):  # noqa: FBT0
                 child_class.objects.filter(featured=True).update(featured=False)
         super().save(clean=clean, user=user, log_action=log_action, **kwargs)
 
+    api_fields = [
+        APIField("child_pages", serializer=ProductChildPageSerializer()),
+    ]
+
```

Comment:
Current API does not return the custom page fields like: subhead etc. We should return the custom page data + child page fields as well.